### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by leonorastiernborg**
Database connection and get request from db

### CODE REVIEW

File: robosistance/Cargo.toml [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 rand = "0.8.5"
 rocket = {version = "0.5.0-rc.3", features = ["json", "uuid"]}
 serde = {version = "1.0.160", features = ["derive"]}
 uuid = {version = "0.7", features = ["serde", "v4"]}
+dotenv = "0.15.0"
+
+[dependencies.mongodb]
```
Comment (Kax-y) [inline robosistance/Cargo.toml]: You can write this on one line if you wish

File: robosistance/src/models/robot_model.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+use mongodb::bson::oid::ObjectId;
```
Comment (Kax-y) [inline robosistance/src/models/robot_model.rs]: I think this file would fit fine in the db folder, I don't think you need another one.

File: robosistance/src/endpoints/db.rs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+use crate::{models::robot_model::RobotData, db::mongodb_robot::MongoRepo};
+use mongodb::{bson::oid::ObjectId, results::InsertOneResult};
+use rocket::{http::Status, serde::json::Json, State};
+use rocket::get;
+
+#[get("/get_robot_data")]
+pub fn get_robot_data(db: &State<MongoRepo>) -> Result<Json<Vec<RobotData>>, Status> {
+    println!("COMES HERE");
```
Comment (Kax-y) [inline robosistance/src/endpoints/db.rs]: Remove this print. You could add an appropriate log if you wish but I don't think it's needed. This seems to be for testing only.

**Review by Kax-y — CHANGES_REQUESTED** [review]
It looks good but just needs some minor changes!
(mostly moving models into the db folder)

**Review by Kax-y — APPROVED** [review]
Looks good!

**Review by 00oskpet — APPROVED** [review]

**Pull request merged**